app/src/main/java/com/example/fbufinalapp/EditDestinationActivity.java

```diff
@@ -103,9 +103,11 @@ protected void onCreate(Bundle savedInstanceState) {
                         SimpleDateFormat timeFormatter = new SimpleDateFormat("hh:mm");
                         SimpleDateFormat amPMFormatter = new SimpleDateFormat("a");
 
-                        binding.etDateSelect.setText(dateFormatter.format(date));
+                        dateSelected = dateFormatter.format(date);
+                        timeSelected = amPMFormatter.format(date);
+                        binding.etDateSelect.setHint(dateSelected);
                         binding.etTime.setText(timeFormatter.format(date));
-                        binding.etTimeSelect.setText(amPMFormatter.format(date));
+                        binding.etTimeSelect.setHint(timeSelected);
```

Comment:
I thought the hint was removed? 
curious, how is `setHint` different from `setText`?

Reply:
Ah I probably should've been more clear! I meant that the default original hint for binding.etDateSelect used to be "Select a date." But having this hint and then trying to set another hint caused the two texts to overlap each other. So I removed the default hint in favor of setting the new values while editing the destination. 

SetHint and setText both change the text of a View, but hints are slightly lighter than text; they show up as gray in the View. They're usually used as prompts for the user, I think!